DisguiseUnityRenderStream/Runtime/DisguiseRenderStream.cs

```diff
@@ -87,6 +87,11 @@ protected DisguiseRenderStream(ManagedSchema schema)
 
         void OnSceneLoaded(Scene loadedScene, LoadSceneMode mode)
         {
+            if (DisguiseRenderStreamSettings.GetOrCreateSettings().exposePresenter)
+            {
+                GameObject.Instantiate(DisguisePresenter.LoadPrefab());
```

Comment:
Curious: Why do we need a prefab? Why not create the components and set their fields via scripting? I can't really review a `.prefab`.

Reply:
Yes the prefab is equivalent to
```csharp
var root = new GameObject("Disguise Presenter");
var disguisePresenter = root.AddComponent<DisguisePresenter>();

var outputPresenterGO = new GameObject("Output Presenter");
outputPresenterGO.transform.parent = root.transform;
var outputPresenter = outputPresenterGO.AddComponent<CameraCapturePresenter>();

var inputPresenterGO = new GameObject("Input Presenter");
inputPresenterGO.transform.parent = root.transform;
var inputPresenter = inputPresenterGO.AddComponent<Presenter>();

disguisePresenter.OutputPresenter = outputPresenter;
disguisePresenter.InputPresenter = inputPresenter;
```

But I can't add the remote parameters through code. They look like this in the prefab:
![image](https://user-images.githubusercontent.com/84032434/228627002-a2bc2b8a-fa3b-4b36-ac4b-0bbda24ea7d6.png)